AI-and-Analytics/Getting-Started-Samples/INC-Quantization-Sample-for-PyTorch/README.MD

```diff
@@ -51,58 +40,86 @@ The sample contains one Jupyter Notebook and one Python script.
+>**Note**: Before running the sample, make sure [Environment Setup](https://github.com/oneapi-src/oneAPI-samples/tree/master/AI-and-Analytics/Getting-Started-Samples/INC-Quantization-Sample-for-PyTorch#environment-setup) is completed.
+
+Go to the section which corresponds to the installation method chosen in [AI Tools Selector](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-tools-selector.html) to see relevant instructions:
+* [Conda/PIP](https://github.com/oneapi-src/oneAPI-samples/tree/master/AI-and-Analytics/Getting-Started-Samples/INC-Quantization-Sample-for-PyTorch#condapip) 
+* [Docker](https://github.com/oneapi-src/oneAPI-samples/tree/master/AI-and-Analytics/Getting-Started-Samples/INC-Quantization-Sample-for-PyTorch#docker)
+* [AI Tools Offline Installer (Validated)](https://github.com/oneapi-src/oneAPI-samples/tree/master/AI-and-Analytics/Getting-Started-Samples/INC-Quantization-Sample-for-PyTorch#ai-tools-offline-installer-validated)
+  
+### Conda/PIP
+> **Note**: Make sure your Conda/Python environment with AI Tools installed is activated
+1. Clone the GitHub repository:
+``` 
+git clone https://github.com/oneapi-src/oneAPI-samples.git
+cd oneapi-samples/AI-and-Analytics/Getting-Started-Samples
+```
+2. Launch Jupyter Notebook: 
+```
+jupyter notebook --ip=0.0.0.0
+```
+4. Follow the instructions to open the URL with the token in your browser.
+5. Select the Notebook:
+```
+optimize_pytorch_models_with_ipex.ipynb
+```
+6. Run every cell in the Notebook in sequence.
+
+### Docker
+AI Tools Docker images already have Get Started samples pre-installed. Refer to [Working with Preset Containers](https://github.com/intel/ai-containers/tree/main/preset) to learn how to run the docker and samples.
+### AI Tools Offline Installer (Validated)  
+1. If you have not already done so, complete [Configure you system](https://www.intel.com/content/www/us/en/docs/oneapi-ai-analytics-toolkit/get-started-guide-linux/2024-0/before-you-begin.html) section from AI Tools Get Started Guide.
+2. Activate the Conda environment:
+```
+conda activate pytorch
+``` 
+3. Clone the GitHub repository:
+``` 
+git clone https://github.com/oneapi-src/oneAPI-samples.git
+cd oneapi-samples/AI-and-Analytics/Getting-Started-Samples
+```
+4. Launch Jupyter Notebook: 
+```
+jupyter notebook --ip=0.0.0.0
+```
+5. Follow the instructions to open the URL with the token in your browser.
+6. Select the Notebook:
+```
+optimize_pytorch_models_with_ipex.ipynb
+```
+7. Change the kernel to `pytorch`
```

Comment:
you might need to register pytorch conda kernel to jupyter kernel first.
https://github.com/IntelAI/models/tree/master/docs/notebooks/perf_analysis#option-1-conda-environment-creation